apps/server/src/main.ts

```diff
@@ -31,6 +31,8 @@ async function bootstrap() {
 	const nestExpressAdapter = new ExpressAdapter(nestExpress);
 	const nestApp = await NestFactory.create(ServerModule, nestExpressAdapter);
 
+	// customize nest app settings
+	nestApp.enableCors();
```

Comment:
There is a reason why CORS is blocked by default in Express: enabling it for all origins is considered a security vulnerability. I know we use JWT, but disabling CORS adds an extra layer of security. That is why I suggest  enabling CORS  only in **local** environment and keep disabled for production. 

Here is a little read on that: https://www.pivotpointsecurity.com/blog/cross-origin-resource-sharing-security/

Reply:
i would add this via flag that in development mode it will be enabled only.

Reply:
That sounds like a good idea to me!

Reply:
```
.options('*', cors())
.use(cors())
```
Is like we have it in the old app. If we use it for the new, it make no different. It sounds like, we need it for production too. Because our routing setup on staging systems is different to live and no good example to test the cors settings. @mamutmk5

Reply:
In production frontend seems to be the same origin, so it's not an issue there, right @mamutmk5 ?

Reply:
That is also the reason why everything works in production.

Reply:
Thats the other way we can solve it and let nuxt proxy the server like in production. nuxt/api->server.
The way in the old app is in terms of security not a solution we should select as default.

Reply:
![image](https://user-images.githubusercontent.com/4550014/129679069-7c9b8790-903d-4311-aa24-a59a6bc68028.png)

The current solution adds the required * we need for the production system and it solves the local issue.

Reply:
We don't need CORS for production system, because frontend is on the same origin:

![Bildschirmfoto 2021-08-17 um 09 07 06](https://user-images.githubusercontent.com/34917508/129680005-138e3160-3b37-4211-b33d-4c4624519c01.png)

Reply:
Please, double check with Paul! But that's what I remember from yesterday.